src/lib/forms/rjsf/widgets/RJSFVocabulary.js

```diff
@@ -133,6 +136,7 @@ RJSFVocabulary.propTypes = {
   value: PropTypes.string,
   options: PropTypes.shape({
     vocabularyType: PropTypes.string.isRequired,
+    size: PropTypes.number,
```

Comment:
missing default ?

Reply:
discuss IRL: options is an object so the size will not be defaulted as we pass as `options: {vocabularyType: '...'}`.